serverless-graphql-api/javascript-appsync-sam-js/README.md

```diff
@@ -0,0 +1,142 @@
+
+The application uses a shared Amazon Cognito stack for authentication/authorization. The shared stack will be deployed automatically if you use the CI/CD pipeline included in this project. If you deploy the application manually, you will need to create this stack and update the `template.yaml` parameters section with the stack name. See the manual deployment section for more details.
+
+## Deploy the CI/CD pipeline
+To create the CI/CD pipeline we will copy the code from the serverless-samples directory into a new and separate directory and use this new directory as the codebase for your pipeline. 
+
+First, navigate to the root directory of this repository. To verify your location, run the command *basename "$PWD"* - your terminal should return `serverless-samples` as an output. Now run the following commands:
+
+```console
+serverless-samples:~$ git subtree split -P serverless-graphql-api -b serverless-graphql-api
+serverless-samples:~$ mkdir ../serverless-graphql-api-cicd && cd ../serverless-graphql-api-cicd
+serverless-graphql-api-cicd:~$ git init -b main
+serverless-graphql-api-cicd:~$ git pull ../serverless-samples serverless-graphql-api
+serverless-graphql-api-cicd:~$ cd javascript-appsync-sam-js
+```
+
+To create the pipeline run the following command:
+
+```console
+javascript-appsync-sam-js:~$ aws cloudformation create-stack --stack-name serverless-api-pipeline --template-body file://pipeline.yaml --capabilities CAPABILITY_IAM
+```
+The pipeline will attempt to run and will fail at the SourceCodeRepo stage as there is no code in the AWS CodeCommit yet.
+
+***NOTE:** If you change stack name, avoid stack names longer than 25 characters. In case you need longer stack names check comments in the pipeline.yaml and update accordingly.*
+
+***Note:** You may need to set up AWS CodeCommit repository access for HTTPS users [using Git credentials](https://docs.aws.amazon.com/codecommit/latest/userguide/setting-up-gc.html?icmpid=docs_acc_console_connect_np) and [set up the AWS CLI Credential Helper](https://docs.aws.amazon.com/console/codecommit/connect-tc-alert-np).*
+
+Once you have access to the code repository, navigate to the javascript-appsync-sam-js folder, and if you changed stack name, make sure that Parameters section of template.yaml is updated with the output values from the shared Amazon Cognito stack, and push the code base to CodeCommit to start an automated deployment:
+
+```console
+javascript-appsync-sam-js:~$ git remote add origin <URL to AWS CodeCommit repository>
+javascript-appsync-sam-js:~$ git push origin main
+```
+
+Navigate to the AWS CodePipeline in the AWS Management Console and release this change if needed by clicking "Release change" button.
+
+![CodePipeline](./assets/CodePipeline.png)
+
+Note that the same Amazon Cognito stack is used in both testing and production deployment stages, and the same user credentials can be used for testing and API access.
+
```

Comment:
Suggest you add some copy about the Manual Approval process.

Reply:
Suggest you add some copy asking me to verify something. For example, when I see this thing, I'm done. Tell me what I've accomplished. Maybe suggest I inspect something to learn about what I've done. Congratulate me. Make sure I understand that I'm complete here and that the next part is an alternate pathway I can go down. Why might I want to go down that pathway?

Reply:
Could you make similar modifications to the README's in the two other directories if relevant?

Reply:
Updated instructions with walkthrough of what was created, screenshots, references to instructions on how to create API client, note on manual deployment.